webpack.config.js

```diff
@@ -23,7 +24,7 @@ module.exports = {
     new HtmlWebpackPlugin({
       inject: false,
       template: require("html-webpack-template"),
-      appMountId: "root"
-    })
-  ]
-}
\ No newline at end of file
+      appMountId: "root",
```

Comment:
is there a rule we have in place to enforce these trailing commas?

Reply:
there is as part of the `prettier` eslint rules as well as it being the only thing configured in prettier 😂 

https://github.com/MichaelQ94/rcubed/blob/a8c605cd0d59f342f54270a4b6c6929a9fcc0647/.prettierrc#L2